jboss_wildfly/assets/dashboards/jboss_wildfly.json

```diff
@@ -0,0 +1,854 @@
+                "font_size": "14",
+                "show_tick": true,
+                "text_align": "left",
+                "tick_edge": "right",
+                "tick_pos": "50%",
+                "type": "note"
+            },
+            "id": 5021725266236704,
+            "layout": {
+                "height": 11,
+                "width": 30,
+                "x": 1,
+                "y": 52
+            }
+        },
+        {
+            "definition": {
+                "background_color": "gray",
+                "content": "A sudden spike in garbage collection count could indicate a memory leak.\n",
+                "font_size": "14",
+                "show_tick": true,
+                "text_align": "left",
+                "tick_edge": "right",
+                "tick_pos": "50%",
+                "type": "note"
+            },
+            "id": 5445132788287032,
+            "layout": {
+                "height": 7,
+                "width": 30,
+                "x": 1,
+                "y": 69
+            }
+        },
+        {
+            "definition": {
+                "background_color": "gray",
+                "content": "Listeners are responsible for translating requests to responses to clients. \n\nThis section can be used to monitor the performance of your listeners and understand the throughput and processing times for the server.",
+                "font_size": "14",
+                "show_tick": true,
+                "text_align": "left",
+                "tick_edge": "left",
+                "tick_pos": "50%",
+                "type": "note"
+            },
+            "id": 5322338467237458,
+            "layout": {
+                "height": 13,
+                "width": 32,
+                "x": 177,
+                "y": 52
+            }
+        },
+        {
+            "definition": {
+                "columns": [
+                    "host",
+                    "service"
+                ],
+                "indexes": [],
+                "message_display": "expanded-md",
+                "query": "source:jboss_wildfly",
+                "show_date_column": true,
+                "show_message_column": true,
+                "sort": {
+                    "column": "time",
+                    "order": "desc"
+                },
+                "title": "",
+                "title_align": "left",
+                "title_size": "16",
+                "type": "log_stream"
+            },
+            "id": 4331835511855062,
+            "layout": {
+                "height": 36,
+                "width": 47,
+                "x": 177,
+                "y": 13
+            }
+        },
+        {
+            "definition": {
+                "background_color": "blue",
+                "content": "Logs",
```

Comment:
Small nit with the `Logs` section. Could you move it to be be aligned with `Transaction Summary` section in height?